cves/CVE-2013-0844.yml

```
@@ -146,10 +149,10 @@ unit_tested:
     In FFmpeg, they have the FATE suite of regression tests. They are not
     "unit" tests per se, but are automated regression tests. If the fix for the
     vulnerability involved adding or updating a FATE suite, then make fix: true.
-  code:
-  code_answer:
-  fix:
-  fix_answer:
+  code: false
+  code_answer: "There was no code coverage for the original vulenrability"
```

```suggestion
  code_answer: There was no code coverage for the original vulnerability.
```
Fields don't need to be wrapped in quotes.

---

Notes